cdk/lib/mobile-save-for-later.ts

```diff
@@ -99,7 +99,13 @@ export class MobileSaveForLater extends GuStack {
     const saveForLaterApi = new GuApiGatewayWithLambdaByPath(this, {
       app,
       restApiName: `${app}-api-${this.stage}`,
-      monitoringConfiguration: { noMonitoring: true }, //TODO: configure monitoring (this is not setup in current CFN)
+      monitoringConfiguration: {
+        snsTopicName: "arn:aws:sns:eu-west-1:201359054765:mobile-server-side",
```

Comment:
You just need the name here (`mobile-server-side`) not the full ARN

Reply:
👍